mozilla_django_oidc_db/config.py

```diff
@@ -109,19 +109,26 @@ def store_config(request: HttpRequest) -> None:
     mozilla-django-oidc's callback view deletes the state key after it has validated it,
     so our :func:`lookup_config` cannot extract it from the session anymore.
     """
-    # The config_class key is added to the state in the OIDCInit.get method.
-    # TODO: verify that the state query param is present for error flows! Need to check
-    # the OAUTH2 spec for this, but according to ChatGeePeeTee if the request contains
-    # it, the callback must have it too.
-    state_key = request.GET.get("state")
-    if not state_key or state_key not in (
-        states := request.session.get("oidc_states", [])
-    ):
-        raise BadRequest("Could not look up the referenced config.")
-
-    state = states[state_key]
+    # Attempt to retrieve the config_class from the session, this only works for users
+    # that are actually logged in as Django users
+    if request.session.get("OIDCDB_CONFIG_CLASS", ""):
```

Comment:
please define a (private) constant for this session key, repeating the magic string without any typing support is going to cause problems at some point.